src/shared.ts

```diff
@@ -21,7 +21,7 @@ export const formatDate = (date: Date): string => {
 
 export const oneDayMs = 1000 * 60 * 60 * 24;
 
-export const getIsTomorrow = (date: Date): boolean => {
+export const getIsWithinDays = (date: Date, days: number): boolean => {
```

Comment:
Nit: Can we call this `getIsWithinFutureDays` since it will return false for yesterday?

Reply:
Done! https://github.com/inkblotty/daily-update-action/pull/39/commits/c4b852676bae9ed7e329c97f62542c8d359fa457